VideoLocker/res/values/dimens.xml

```diff
@@ -59,7 +59,13 @@
     <dimen name="signup_social_title_margin_large">8dp</dimen>
     <dimen name="signup_social_title_margin_bottom">1dp</dimen>
 
+    <dimen name="edx_hairline">1px</dimen>
```

Comment:
Do we really want to define this in pixels? It would be extremely thin in high resolution devices...